### ISSUE TITLE
Possible confusing explanations in sections 2.5 and 2.6.

### ISSUE THREAD

**Issue by nbro** [ermongroup/cs228-notes#162]
In [section 2.5](https://ermongroup.github.io/cs228-notes/preliminaries/probabilityreview/), you do not explain what **I** is supposed to be, but it might be necessary to explain it, as not everyone might be familiar with this indicator function.

In [section 2.6](https://ermongroup.github.io/cs228-notes/preliminaries/probabilityreview/), you describe the Bernoulli random variable as follows

> one if a coin with _heads probability p_ comes up heads, zero otherwise.

Specifically, the expression "heads probability" is a little confusing. You should have been more explicit "with probability p of getting heads".

It could be formulated as (or something like that)

> this random variable takes the value p, if the input to the random variable is X=(x=1) (or x=heads), and it takes the value 1 - p, if X=(x=0) (or x=tails). In other words, the probability of taking the value "heads" (1) is p and the probability of taking the value "tails" (0) is 1 - p.

At least, you I would rephrase your explanations to make them less ambiguous and confusing.

I think that the explanations of the other random variables are also quite poor. For example, the geometric random variable is quite related to the binomial random variable, but you do not explain it. See: https://stats.stackexchange.com/q/263141/82135. In general, I think you should spend 2-3 sentences to motivate better the use of these particular random variables. This would make your notes even more useful!

**Comment by chrisyeh96** [comment]
**Indicator function:** Good suggestion. Added in commit 3680ac.

**Bernoulli R.V.**: also revised. However, your formulation is incorrect - the random variable does not take the value p. It takes the value 0 or 1.

**Explanations of other distributions** - it is great that you are able to make connections between the different distributions. However, as this course is focused on PGMs and not statistics, we do not go in depth into the different distributions here.
